### PR TITLE
Improved vtk time step exporter

### PR DESCRIPTION
**Pull request by keileg**
The exporter used the time step both to count time (as displayed by paraview) and to extend file names to differ between time steps. The latter required the time step to be an int. We remove this, let the file extension by default run from 0 (but can be overridden), and set the time as displayed by paraview when exporting the vtk.

The setup works on my local setup (no unit tests for vtk are run on travis).

### CODE REVIEW

File: src/porepy/viz/exporter.py [hunk 1]
```diff
@@ -310,19 +333,13 @@ def write_pvd(self, time):
...
-
         if self.is_GridBucket:
-            [
-                o_file.write(fm % (time[t], self._make_file_name(self.name, t, dim)))
-                for t in time_step
-                for dim in self.dims
-            ]
+            for time, fn in zip(timestep, file_extension):
+                for dim in self.dims:
+                    o_file.write(fm % (time, self._make_file_name(self.name, fn, dim)))
```
Comment (alessiofumagalli) [inline src/porepy/viz/exporter.py]: more readable :)

**Comment by alessiofumagalli** [comment]
It looks ok, is this part mandatory or we can still use what we already have?

File: src/porepy/viz/exporter.py (hunk 1 above)
Comment (keileg) [reply src/porepy/viz/exporter.py]: Indeed.

**Comment by keileg** [comment]
Yes, the old setup should still work.

**Pull request merged**